index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <main id="main">
+        <div class="header">
+                    <H1 id="title">Dr. Norman Borlaug</H1>
+                    <p>The man who saved a billion lives</p>
+                    </div>
+                    <div id="img-div">
+                        <img id="image" class="center" 
+                        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+                   <figcaption id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+                    </div>
+                    <div id="tribute-info">
+                        <h2 id="subtitle">Here's a time line of Dr. Borlaug's life:</h2>
+                    </div>
+                    <div>
+                        <ul><li><span>1914</span>- Born in Cresco, Iowa</li>
+                        <li><span>1933</span> Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"</li>
+                        <li><span>1931</span>- Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me. </li>
+                        <li><span>1937</span>- Finishes university and takes a job in the US Forestry Service</li>
```

Comment:
It would be better to arrange your code. because it will help other developers to review your project.